pyqt-apps/siriushla/li_ap_mps/main.py

```diff
@@ -0,0 +1,533 @@
+        if title == 'Klystrons':
+            count = self.countGen(count, 4)
+        elif title == 'Water':
+            count = self.countWater(count)
+        elif title == 'Modulator':
+            count = self.countGen(count, 2)
+        elif title == 'Gate Valve':
+            count = self.countGen(count, 5)
+        elif title in ['Temperature', 'Water Temperature']:
+            self.countTemp(count, title)
+        elif title == 'VA':
+            count = self.countVA(count)
+        else:
+            count = self.countGen(count, 1)
+
+        return count
+
+    def setTitleLabel(self, item, axis, layout):
+        ''' Display title labels '''
+        pos = [0, 0]
+        for pos[axis] in range(1, len(item)+1):
+            lbl_header = QLabel('<h4>'+item[pos[axis]-1]+'</h4>')
+            lbl_header.setAlignment(Qt.AlignCenter)
+            layout.addWidget(lbl_header, pos[0], pos[1], 1, 1)
+            if axis == 0:
+                lbl_header.setMaximumWidth(50)
+        return layout
+
+    def setPvLbl(self, pv_name):
+        ''' Display the water title label'''
+        lbl_pv = LBL_WATER.get(pv_name)
+        lbl_title_wid = QLabel(lbl_pv)
+        lbl_title_wid.setAlignment(Qt.AlignCenter)
+        return lbl_title_wid
+
+    def setParamLabel(self, layout):
+        ''' Display Temperature parameters labels '''
+        pos = 0
+        for times in range(0, 2):
+            for item in TEMP_TYPE:
+                lbl_param = QLabel(item)
+                lbl_param.setAlignment(Qt.AlignCenter)
+                layout.addWidget(lbl_param, 1, pos, 1, 1)
+                pos += 1
+        return layout
+
+    def setTempHeader(self, item_list, layout):
+        ''' Display Temperature header labels '''
+        widget = QWidget()
+        hd_glay = QGridLayout()
+        pos = 0
+        for item in item_list:
+            lbl_header = QLabel('<h4>'+item+'</h4>')
+            lbl_header.setAlignment(Qt.AlignCenter)
+            hd_glay.addWidget(lbl_header, 0, pos, 1, 3)
+            pos += 3
+
+        hd_glay = self.setParamLabel(hd_glay)
+        widget.setLayout(hd_glay)
+        layout.addWidget(widget, 0, 1, 1, 6)
+        return layout
+
+    def getSingleTitle(self, title, layout):
+        ''' Display a single Title '''
+        if title in LBL_MPS:
+            lbl_item = LBL_MPS.get(title)
+            layout = self.setTitleLabel(lbl_item[0], 0, layout)
+            if title not in ['Water Temperature', 'Temperature']:
+                layout = self.setTitleLabel(lbl_item[1], 1, layout)
+            else:
+                layout = self.setTempHeader(lbl_item[1], layout)
+        return layout
+
+    def getParamAll(self, has_title, pv_name, wid_type):
+        if wid_type == 0 or pv_name in ['PPState7',
+                'Mod1State', 'GPS1', 'WaterState', 'K1PsState',
+                'IP1Warn', 'CCG1Warn', 'CCG1Alarm', 'PRG1Warn',
+                'WFS1', 'WFS2', 'WFS3', 'WFS4', 'WFS5', 'WFS6']:
```

Comment:
I think it is better to create a variable with this list and use it in the if statement